djangocms_text_ckeditor/static/djangocms_text_ckeditor/ckeditor_plugins/cmsplugins/plugin.js

```diff
@@ -13,7 +13,7 @@
      * @returns {String} div|span
      */
     function getFakePluginElement(pluginMarkup) {
-        var innerTags = (pluginMarkup.match(/<([\S]*?)\s[\s\S]*?>/) || [0, false]).splice(1);
+        var innerTags = (pluginMarkup.match(/<\s*([^>\s]+)[\s\S]*?>/) || [0, false]).splice(1);
```

Comment:
That doesn't seem to be equivalent. Using the code from the example I gave in the report on #415, that returns `"h"` rather than the desired `"h1"`. The fact that the `[\s\S]*?` part is happy to match everything until the end of the tag means that it never backtracks to the `(\S+?)` to get more characters.